common/culture/cultures/goblin.txt

```diff
@@ -55,10 +55,7 @@
 		mother_name_chance = 5
```

Comment:
Mostly goblins look good in western clothes, but female common headgears don't fit them.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/108849868-16396280-75fc-11eb-8c3b-1fd70561dc90.png)

</details>

Reply:
> 
> 
> Mostly goblins look good in western clothes, but female common headgears don't fit them.
> Click to expand

I haven't made any changes to their clothing yet, they're just using whatever was assigned to their culture beforehand. That said, noted.

Reply:
Fixed in 3a33cac785cac51ae251a808b1b0251bf7ecc61b!